packages/cli/lang/en.lyaml

```diff
@@ -528,6 +528,7 @@ en:
             invalidAccountTypeDescription: "Only public apps created in a developer account can be converted to a project component. Select a connected developer account with {{useCommand}} or {{authCommand}} and try again."
             projectAlreadyExists: "A project with name {{ projectName }} already exists. Please choose another name."
             invalidAppId: "[--appId] Could not find appId {{ appId }}. Please choose another public app."
+            invalidMarketplaceApp: "[--appId]: Could not migrate appId {{ appId }}. Marketplace apps cannot be migrated at this time. Please choose another public app."
```

Comment:
Do we need to include the `[--appId]` flag here? That's not typically a pattern that we use in our error messages. We normally only show those flag examples inside of the prompt questions themselves as ways for users to discover how to bypass the prompt next time.

Also small nit, but there's a colon in this message and there isn't one in the `invalidAppId` message

Reply:
I can remove this in a separate PR--will put it out Monday.